app-benchmarks/cpuburn/cpuburn-1.4a-r2.ebuild

```diff
@@ -0,0 +1,65 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit eutils
+
+MY_PV="${PV/./_}"
+
+DESCRIPTION="CPU testing utilities in optimized assembler for maximum loading"
+HOMEPAGE="https://downloads.bl4ckb0x.de/pages.sbcglobal.net/redelm/"
+SRC_URI="https://downloads.bl4ckb0x.de/pages.sbcglobal.net/redelm/cpuburn_${MY_PV}_tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~arm ~x86"
+LICENSE="GPL-2"
+SLOT="0"
+
+PATCHES=( "${FILESDIR}/01-variables.patch" )
+
+QA_FLAGS_IGNORED="usr/bin/burnBX
+	usr/bin/burnK6
+	usr/bin/burnK7
+	usr/bin/burnMMX
+	usr/bin/burnP5
+	usr/bin/burnP6"
+
+QA_TEXTRELS="${QA_FLAGS_IGNORED}"
+
+src_prepare() {
+	default
+
+	# Respect users compiler and users CFLAGS and LDFLAGS on x86/amd64
+	# Must be always compiled in 32-bit on amd64 arch
+	# See https://bugs.gentoo.org/65719
+	if ! use arm; then
+		sed -i -e 's/gcc -s/$(CC) $(CFLAGS) -m32 $(LDFLAGS)/' Makefile || die
+	fi
+
+	# Respect users compiler and users CFLAGS and LDFLAGS on arm
+	if use arm; then
+		sed -i -e '/CC :=/d' -e 's/^.*-mfloat-abi=softfp/	$(CC) $(CFLAGS) -nostdlib $(LDFLAGS)/' ARM/Makefile || die
+	fi
+}
+
+src_compile() {
+	if use arm; then
+		cd "${S}"/ARM
```

Comment:
`|| die`

Reply:
Sorry, forgot to add the die. Saw it today early, but forgot to say here, that I will fix it. Done now.